### ISSUE TITLE
builder.js example not working

### ISSUE THREAD

**Issue by wstam88** [romgrk/node-gtk#118]
I cannot seem to get the builder.js example running:

```
win.setDefaultSize(600, 800);
    ^

TypeError: win.setDefaultSize is not a function
    at Object.<anonymous> (/home/nabu/GitHub/gtk-stuff/node-gtk/examples/builder.js:14:5)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
```

Any ideas? Other examples are working.

**Comment by romgrk** [comment]
Yes, it seems that objects retrieved through `builder.getObject(name)` don't have their prototype set to the correct value.
For now, I've fixed the `builder.js` example by setting the prototype directly on all objects.
I've opened #119 to track the underlying issue.

Thanks for the report!

**Comment by wstam88** [comment]
Jup that fixed it!

Btw awesome project! 😎

**Comment by romgrk** [comment]
Thanks :) I patched the issue in #120, so `Gtk.Builder#getObject` now returns objects with correct prototype (though only for the `Gtk` namespace). It's on master and will be in the next release.
